src/ImageSharp/Formats/Jpeg/JpegDecoderCore.cs

```diff
@@ -936,12 +936,18 @@ private ushort ReadUint16()
         private Image<TPixel> PostProcessIntoImage<TPixel>()
             where TPixel : struct, IPixel<TPixel>
         {
+            var image = Image.CreateUninitialized<TPixel>(
```

Comment:
Everywhere, where we can make sure that the operation is about to fill the whole image.
So all decoders. Copying processors: not sure, they are a bit more complicated because of the swapping logic.